src/build/build-libgerbil.ss

```diff
@@ -386,24 +388,31 @@
     (displayln "... build " libgerbil)
     (let (libgerbil-ldd (filter (? (not string-empty?)) (string-split ld-options #\space)))
       (if (eq? mode 'shared)
-        (invoke-gcc ["-shared" "-o" libgerbil
-                     libgerbil-ldd ...
-                     static-module-o-paths ...
-                     builtin-modules-o-path
-                     link-o-path])
+        (cond-expand
+          (darwin (invoke-gcc ["-dynamiclib" "-o" libgerbil "-install_name"
+                               (string-append (getenv "GERBIL_PREFIX") "/lib/libgerbil.dylib")
```

Comment:
you can use `(path-expand "lib/libgerbil.dylib" (getenv "GERBIL_PREFIX"))` for this, probably a little cleaner.

Reply:
👍🏻